plugins/source/aws/resources/services/cloudformation/stacks_fetch.go

```diff
@@ -42,9 +42,6 @@ func fetchCloudformationStackResources(ctx context.Context, meta schema.ClientMe
 	for {
 		output, err := svc.ListStackResources(ctx, &config)
 		if err != nil {
-			if client.IsErrorRegex(err, "ValidationError", validStackNotFoundRegex) {
```

Comment:
Should also remove `validStackNotFoundRegex`?

Reply:
done.